easypy/signals.py

```diff
@@ -399,6 +422,9 @@ def register_object(obj, **kwargs):
         method = getattr(obj, method_name)
         assert callable(method), "'%s' (%r) is not callable" % (method, obj)
 
+        if getattr(method, "_signals_dont_register", False):
```

Comment:
This worries me a bit, because `Foo.on_bar` is not the same as `Foo().on_bar`. While the regular `__get__` and method type pass this, I'm not sure if it is wise to count on it...

Reply:
> If we inherit these methods, how is this going to solve the problem?
I would apply it at the base, or override the methods.

but maybe a better way would be to pass a black-list to `register_object`?
```register_object(obj, ignore=["on_finished", "on_started"])```

Reply:
I'm worried about that too - `register_object` is often used generically, without knowing which object we are registering...

Why not just override these methods?
```python
class Foo:
    def on_bar(self):
        ...


class Baz(Foo)(:
    on_bar = None

    def on_qux(self):
        ...


register_object(Baz())  # will not register `on_bar` because it's not a function.
```